### PR TITLE
Remove man page Markdown source

### PR DESCRIPTION
**Pull request by rhc54**
Requiring that people install pandoc has become quite an issue
and generated some heated exchanges. Avoid this conflict by
moving the man page sources to their own repo, and only including
the rendered nroff here.

Signed-off-by: Ralph Castain <rhc@pmix.org>

### CODE REVIEW

**Comment by rhc54** [comment]
Hmmm....seeing this warning repeated multiple times:
```txt
> [bf75287b0eba:10378]ENCE WARNINGS 4294967295 4294967295
```
Never seen that one before

**Comment by rhc54** [comment]
bot:ibm:prrte:retest

**Comment by rhc54** [comment]
@jjhursey Do you have any idea what these warnings mean?
```
[787d44b45f4e:08443]ENCE WARNINGS 4294967295 4294967295
```

File: src/mca/pstrg/lustre/pstrg_lustre.c [hunk 1]
```diff
@@ -156,6 +156,7 @@ static pmix_status_t query(pmix_query_t queries[], size_t nqueries, pmix_list_t
                 }
             }
 
+            pmix_output(0, "SILENCE WARNINGS %u %u", uid, gid);
```
Comment (jjhursey) [inline src/mca/pstrg/lustre/pstrg_lustre.c:159]: I think the cause of the CI failure is this debugging line that was left in place.

**Pull request merged**